docs/design/coreclr/botr/intro-to-clr.md

```diff
@@ -56,7 +56,8 @@ The runtime has many features, so it is useful to categorize them as follows:
 1. **Fundamental features** – Features that have broad impact on the design of other features.  These include:
     1. Garbage Collection
     2. Memory Safety and Type Safety
-    3. High level support for programming languages.
+    3. Common Intermediate Language to support multiple programming languages
+    4. Just-in-time compiler
```

Comment:
`Just-in-time compiler` is not a fundamental feature. It is an implementation detail, and some CLR form factors do not even include it.

Reply:
That's fair